android/sdk/src/main/java/org/jitsi/meet/sdk/ReactInstanceManagerHolder.java

```diff
@@ -37,7 +37,6 @@
 
 import org.devio.rn.splashscreen.SplashScreenModule;
 import org.webrtc.EglBase;
-import org.webrtc.Logging;
```

Comment:
Keep this import, IIRC it's used in a section which is commented out as an example, but it helps not having to add the import each time.